src/gui/dbsettings/DatabaseSettingsWidgetGeneral.ui

```diff
@@ -190,6 +190,58 @@
         </property>
        </widget>
       </item>
+      <item>
+       <layout class="QHBoxLayout" name="horizontalLayout_2">
+        <item>
+         <widget class="QCheckBox" name="autosaveDelayCheckBox">
+          <property name="toolTip">
+           <string>Autosave delay since last change</string>
+          </property>
+          <property name="accessibleName">
+           <string/>
+          </property>
+          <property name="text">
+           <string>Autosave delay</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QSpinBox" name="autosaveDelaySpinBox">
+          <property name="toolTip">
+           <string>Autosave delay since last change in minutes</string>
+          </property>
+          <property name="accessibleName">
+           <string>Autosave delay since last change in minutes</string>
+          </property>
+          <property name="suffix">
+           <string> min</string>
+          </property>
+          <property name="minimum">
+           <number>0</number>
+          </property>
+          <property name="maximum">
+           <number>420000000</number>
```

Comment:
Hm, the maximum is 291 days, why?

Reply:
I don't really think there is any sense in limiting the user too much, i picked this value to make the UI similar with the other SpinBoxes already existing, the SpinBox changes size with regards to its maximum.
What could be an alternative value?

Reply:
Ok, let's do the same way as another similar examples do.